smartseq2_single_sample/tasks/run_pipeline.wdl

```diff
@@ -0,0 +1,77 @@
+import "star_pipeline.wdl" as run_star
+import "hisat2_pipeline.wdl" as run_hisat2
+import "hisat2_rsem_pipeline.wdl" as run_hisat2_rsem
+import "kallisto_pipeline.wdl" as run_kallisto
+
+workflow run_pipelines {
+
+  ## load annotation
+  File gtffile
+  File genome_ref_fasta
+  File rrna_intervals
+  File gene_ref_flat
+  #load index
+  File star_ref_index
+  File hisat2_ref_index
+  File hisat2_ref_trans_index
+  File rsem_ref_index
+  File kallisto_ref_index
+  ##ref index name
+  File hisat2_ref_name
+  File hsiat2_ref_trans_name
```

Comment:
Is this a typo? Have you tested that this pipeline runs?

Reply:
yes, it is typo but the input json was generated by this file. so it ran through....